django/apps/core/templates/_strossle.sdk.html

```diff
@@ -1,7 +1,8 @@
 <!-- Strossle SDK -->
 <script>
   window.strossle=window.strossle||function(){(strossle.q=strossle.q||[]).push(arguments)};
-  // strossle('dff15dfe-e8ca-4e6d-b547-8038ab88562b', '.strossle-widget');
+  //strossle('dff15dfe-e8ca-4e6d-b547-8038ab88562b', '.strossle-widget');
 </script>
 <script async src="https://assets.strossle.com/strossle-widget-sdk/1/strossle-widget-sdk.js"></script>
+<script src="//widgets.sprinklecontent.com/v2/sprinkle.js <http://widgets.sprinklecontent.com/v2/sprinkle.js> " async></script>
```

Comment:
To get the v1 widget to work properly with react, we have to modify both the script tag here, and the div later, and add a few attributes.

The article is about "infinite scroll", but it's also relevant when we have client side url routing, which universitas has.

https://servicedesk.strossle.com/hc/en-us/articles/360000228029-Does-the-widget-work-with-infinite-scroll-